<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotChatMainUIClass</class>
 <widget class="QWidget" name="RobotChatMainUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>521</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>521</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>521</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RobotChatMainUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QLabel#lb_title{
	font: 20px &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}

QFrame#frame{
	border:1px;
	border-bottom:1px solid #ffffff
}

QPushButton#pb_min{
	border:none;
	background-image: url(:/QtTest/icon/chat/min.png);
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton#pb_close{
	border:none;
	background-image: url(:/QtTest/icon/chat/close.png);
	background-position: center;
    background-repeat: no-repeat;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>580</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_title">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Conversation Suggestion</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>251</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_min">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="WRobotChat" name="chat_widget"/>
     <widget class="WChatDesc" name="chat_desc_wgt"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WRobotChat</class>
   <extends>QWidget</extends>
   <header>WRobotChat.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WChatDesc</class>
   <extends>QWidget</extends>
   <header>WChatDesc.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
